src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -199,17 +199,18 @@ public virtual IReadOnlyList<SarifLog> SplitLogFile(SarifLog sarifLog)
 
                     Logger.LogInformation($"Splitting strategy - {splittingStrategy}");
 
-                    if (splittingStrategy == SplittingStrategy.None)
-                    {
-                        return new[] { sarifLog };
-                    }
-
                     PartitionFunction<string> partitionFunction = null;
 
                     Stopwatch splittingStopwatch = Stopwatch.StartNew();
 
                     switch (splittingStrategy)
                     {
+                        case SplittingStrategy.None:
+                        {
+                            // get ride of results should not be filed
```

Comment:
Remove this comment. First, it does not follow convention, starting with a capital letter, ending with a period. It doesn't read very well grammatically. Not sure it provides useful additional information. Finally, this line of code is identical to other lines in this file, why are they commented? If they don't need to be, why is this line commented?